x/wasm/types/tx.go

```diff
@@ -78,6 +79,16 @@ func (msg MsgStoreCode) GetSigners() []sdk.AccAddress {
 	return []sdk.AccAddress{sdk.WasmToAccAddress(senderAddr)}
 }
 
+func (msg MsgStoreCode) FnSignatureInfo() (string, int, error) {
+	codeLen := len(msg.WASMByteCode)
+	var err error
+	if codeLen <= 0 {
```

Comment:
```suggestion
	if codeLen == 0 {
```

Reply:
**codeLen** must eaual or larger than 0, here is compatibility.